DCA/colors.py

```diff
@@ -0,0 +1,40 @@
+from PIL import Image
+import os
+import argparse
+from pathlib import  Path
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input", type=Path, required=True, help="Input images path.")
+    parser.add_argument("-o", "--output", type=Path, required=True, help="Output images path.")
+    return parser.parse_args()
+
+def color_image():
+    colors = {
```

Comment:
It is worth making a config for the markup configuration. What does tuple key mean?